src/porepy/viz/exporter.py

```diff
@@ -520,9 +518,44 @@ def _from_vector_format(
                             )
 
                             offset += intf.num_cells
+                if key in vtu_data.point_data:
```

Comment:
Reminder for me: What happens if `key` is in `point_data` and `cell_data`? Do the tests cover this case?

Reply:
since cell_data and point_data are two different inputs then if key is present in both it should enter in both parts